pkg/planner/core/find_best_task.go

```diff
@@ -186,6 +187,15 @@ func enumeratePhysicalPlans4Task(
 	if _, ok := p.Self().(*logicalop.LogicalSequence); ok {
 		iteration = iterateChildPlan4LogicalSequence
 	}
+	var fd *fd.FDSet
+	// while for join's both child, anyway and currently we should keep exchanger within the MPP for now
```

Comment:
add a todo here since the fd should be maintained as logical prop instead of constructing it in physical phase.

Reply:
I have add this TODO.